README.md

```diff
@@ -176,18 +197,18 @@ start the server with a docker:
 docker run -d -p 2700:2700 alphacep/kaldi-en:latest
 ```
 
-Then configure the transcription class with the following properly in `~/jigasi/jigasi-home/sip-communicator.properties`:
+Then configure the transcription class with the following property in `~/jigasi/jigasi-home/sip-communicator.properties`:
 
 ```
-org.jitsi.jigasi.transcription.customService=org.jitsi.jigasi.transcription.VoskTranscriptionService
```

Comment:
Will the old one continue to work? As it will break people deployments once they update.